cfy_manager/components/nginx/nginx.py

```diff
@@ -196,7 +196,7 @@ def _replace_external_certs(self):
         self._handle_external_cert(replacing_ca=replacing_ca)
 
     def log_replacing_certificates(self, certs_type):
-        self.logger.info('Replacing %s on nginx component', certs_type)
+        logger.info('Replacing %s on nginx component', certs_type)
```

Comment:
if you want more -lines then just inline these stupid methods

we _really_ don't need a layer for writing a single log message